autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -4,18 +4,89 @@
 from django.core.management import base
 from scraper.banner_requests import BannerRequests
 from scraper.models import Course, Instructor, Section, Meeting
+from course import generate_course_id
```

Comment:
```suggestion
from scraper.models.course import generate_course_id
```

Reply:
I don't know how to run it on VSC but it worked when I copy pasted the output for scrape courses into Spyder and ran the code using that data.

Reply:
You can run `python manage.py scrape_courses` in the terminal to run it in VSCode. Let me know on Slack if you have any questions about setting anything up